README.md

```diff
@@ -661,6 +662,48 @@ if (user.firstName && user.lastname) {
 }
 ```
 
+### 6.4 Avoid multiple if's
```

Comment:
**suggestion:** add the anchor link please
```suggestion
<a name="avoid-multiple-ifs"></a>

### 6.4 Avoid multiple if's
```

Reply:
Great!